tensorflow-xla-runtime.spec

```diff
@@ -30,7 +30,7 @@ pushd tensorflow/xla_aot_runtime_src
     -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
     -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} \
     -DCMAKE_PREFIX_PATH=${ABSEIL_CPP_ROOT} \
-    -DLIBFFT_WRAPPER=${TENSORFLOW_ROOT}/lib/libfft_wrapper.so \
+    -DCMAKE_LINKER_FLAG=-L${TENSORFLOW_ROOT}/lib -lfft -lfft_wrapper
```

Comment:
you might want to use double quotes here `-DCMAKE_LINKER_FLAG="-L${TENSORFLOW_ROOT}/lib -lfft -lfft_wrapper"`

Reply:
and missing `\` at the end

Reply:
also double check the correct cmake flag name for linker flags